hts-precompile/IHederaTokenService.sol

```diff
@@ -441,6 +565,186 @@ interface IHederaTokenService {
+        external
+        returns (int responseCode, NonFungibleTokenInfo memory nonFungibleTokenInfo);
+
+    /// Operation to freeze token account
+    /// @param token The token address
+    /// @param account The account address to be frozen
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function freezeToken(address token, address account)
+        external
+        returns (int responseCode);
+
+    /// Operation to unfreeze token account
+    /// @param token The token address
+    /// @param account The account address to be unfrozen
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function unFreezeToken(address token, address account)
+        external
+        returns (int responseCode);
+
+    /// Operation to grant kyc to token account
+    /// @param token The token address
+    /// @param account The account address to grant kyc
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function grantTokenKyc(address token, address account)
+        external
+        returns (int responseCode);
+
+    /// Operation to revoke kyc to token account
+    /// @param token The token address
+    /// @param account The account address to revoke kyc
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function revokeTokenKyc(address token, address account)
+        external
+        returns (int responseCode);
+
+    /// Operation to pause token
+    /// @param token The token address to be paused
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function pauseToken(address token) external returns (int responseCode);
+
+    /// Operation to unpause token
+    /// @param token The token address to be unpaused
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function unPauseToken(address token) external returns (int responseCode);
+
+    /// Operation to wipe fungible tokens from account
+    /// @param token The token address
+    /// @param account The account address to revoke kyc
+    /// @param amount The number of tokens to wipe
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function wipeTokenAccount(
+        address token,
+        address account,
+        uint32 amount
+    ) external returns (int responseCode);
+
+    /// Operation to wipe non fungible tokens from account
+    /// @param token The token address
+    /// @param account The account address to revoke kyc
+    /// @param  serialNumbers The serial numbers of token to wipe
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function wipeTokenAccount(
+        address token,
+        address account,
+        uint64[] memory serialNumbers
```

Comment:
I thing here should be int64[] to be compliant with already implemented precompiles(burn, transferNFTs and etc.).